tests/test/test_rks_administration_api.py

```diff
@@ -141,6 +141,122 @@ def test_create_group(self, admin_api):
         response, status, headers = admin_api.delete_group_with_http_info("fakecdn1")
         assert status == 204, "Delete group does not return 204 as expected"
 
+    def test_update_group(self, admin_api, group_token):
+        """Test case for update_group
+
+        Update a Group of Nodes to update registration information
+        """
+
+        _, status, headers = admin_api.update_group_with_http_info(
+            "fakecdn1", utils.group_reg_info_updated
+        )
+        assert status == 200
+
+        group_reg_info, status, headers = admin_api.get_group_with_http_info("fakecdn1")
+        assert (
+            group_reg_info.callback_url,
+            group_reg_info.oauth_url,
+            group_reg_info.oauth_client_id,
+            group_reg_info.oauth_client_secret,
+        ) == (
+            utils.group_reg_info_updated.callback_url,
+            utils.group_reg_info_updated.oauth_url,
+            utils.group_reg_info_updated.oauth_client_id,
+            utils.group_reg_info_updated.oauth_client_secret,
+        ), "Group registration information not well updated"
```

Comment:
```suggestion
        assert group_reg_info == utils.group_reg_info_updated, "Group registration information not well updated"
```

The GroupRegInfo class has an `__eq__` method which is used when comparing 2 GroupRegInfo objects: 

```python
    def __eq__(self, other):
        """Returns true if both objects are equal"""
        if not isinstance(other, GroupRegInfo):
            return False

        return self.__dict__ == other.__dict__
```
Internally it checks if the 2 dicts are the same
Dict also implement `__eq__` method (https://stackoverflow.com/a/40921229)